frontend/src/api/ApiCaller.tsx

```diff
@@ -125,6 +126,27 @@ export class BackendAPICaller {
         })
         return result.body
     }
+
+    async pauseMission(robotId: string) {
+        const path: string = 'robots/' + robotId + '/pause'
+        await this.POST(path, '').catch((e) => {
+            throw new Error(`Failed to POST /${path}: ` + e)
```

Comment:
If the control button post fails, should we force an update of the mission status? For instance if pause fail, then the pause button should be the option rather than the play button?

Reply:
This could also be related to #381, but require a faster update rate

Reply:
Will be fixed by having the frontend show the true backend mission state, as described in #386